rust/src/nfs/nfs.rs

```diff
@@ -1359,7 +1363,7 @@ impl NFSState {
     }
     fn setfileflags(&mut self, direction: u8, flags: u16) {
         SCLogDebug!("direction: {}, flags: {}", direction, flags);
-        if direction == 1 {
+        if direction == 1 {        // TODO change toclient
```

Comment:
Resolve TODO

Reply:
@victorjulien is this check correct? Or should it be `if direction  == STREAM_TOCLIENT`? JFYI, `STREAM_TOCLIENT` is 0x08.